pkg/apis/camel/v1/trait/environment.go

```diff
@@ -29,6 +29,7 @@ type EnvironmentTrait struct {
 	HTTPProxy *bool `property:"http-proxy" json:"httpProxy,omitempty"`
 	// A list of environment variables to be added to the integration container.
 	// The syntax is KEY=VALUE, e.g., `MY_VAR="my value"`.
+	// The value may also be a reference to a configmap or secret, e.g. `MY_VAR=configmap:my-cm/my-cm-key`
```

Comment:
You need to perform `make generate` to regen the spec and documentation.

Reply:
I unfortunately get an NPE when running this on my Mac as well as on some Linux server

Reply:
Okey, we have an auto regen in the nightly so it will be eventually sync-ed, but you should report this to get it sorted.